src/http-common.js

```diff
@@ -1,7 +1,7 @@
 import axios from "axios";
 
 export default axios.create({
-  baseURL: "http://localhost:8000",
+  baseURL: "http://localhost:6969",
```

Comment:
start removing axios from everywhere
backend runs on port 6969 now to avoide conflicts